README.md

```diff
@@ -70,4 +70,49 @@ To make use of a built image, run:
         docker pull docker.montagu.dide.ic.ac.uk:5000/montagu-reporting-api:master
         docker run --rm -p 8080:8080 -v {PATH_TO_ORDERLY}:/orderly docker.montagu.dide.ic.ac.uk:5000/montagu-reporting-api:master
 
-replacing `{PATH_TO_OPRDERLY}` with an absolute path to an Orderly directory.
+replacing `{PATH_TO_ORDERLY}` with an absolute path to an Orderly directory.
+
+
+## Orderly Web database schema
+
+Tables required by Orderly Web, relating to users and permissions, are added to the Orderly database by this application. Code and migrations for 
```

Comment:
The "schema" will relate to more stuff than users and perms (next will be publish status) so maybe just say "relating to presentation and access logic, e.g. users & permissions"?